### PR TITLE
Display a message in the env switcher if there are no models

### LINKED ISSUE #2074: Discuss the model switcher empty state
@hatched @urosj Do you think that the current empty state is correct? It currently shows a button to the profile, or should this be a 'new' button which creates an empty canvas?

### LINKED ISSUE #2075: Model switcher should not be transparent
Please can the model switcher be a solid colour.

image

### PR DESCRIPTION
**Pull request by huwshimi**
Fixes #2074 and fixes #2075.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- open the gui when you have no models
- check that the model switcher in the header displays a message when open
- add an app
- drag it under the model switcher
- the switcher should be opaque

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6016/
Test PASSed.

File: jujugui/static/gui/src/app/components/env-switcher/list/list.js [hunk 1]
```diff
@@ -40,7 +40,14 @@ YUI.add('env-list', function() {
 
     generateEnvList: function() {
       var envs = [];
-      this.state.envs.forEach(function(env) {
+      const envList = this.state.envs;
+      if (envList.length === 0) {
+        return (<li className="env-list__environment"
+          key="none">
+          No models
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/env-switcher/list/list.js]: "No models available, click below to view your profile and create a new model" 

?

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6029/
Test PASSed.

**Comment by hatched** [comment]
Thanks @huwshimi :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**